indico/modules/events/abstracts/templates/display/abstracts.html

```diff
@@ -0,0 +1,77 @@
+{% extends 'layout/conference_page_base.html' %}
+{% from 'events/abstracts/management/_abstract_list.html' import render_abstract_list %}
+{% from 'events/management/_lists.html' import render_displayed_entries_fragment %}
+
+{% block title %}
+    {% trans track_title=track.title -%}
+        Abstracts from <span class="track-name">{{ track_title }}</span>
+    {%- endtrans %}
+{% endblock %}
+
+{% block content %}
+    {% if not abstracts %}
+        <p>
+            {%- trans -%}
+                There are currently no abstracts within this track which you are reviewer or convener of.
```

Comment:
There are currently no abstracts within this track **for** which you are **a** reviewer or **a** convener ~~of~~. Isn't this better? I'm asking for opinions.